database_admin/schema/create_schema.sql

```diff
@@ -620,6 +620,7 @@ CREATE TABLE IF NOT EXISTS baseline
     rh_account_id INT               NOT NULL REFERENCES rh_account (id),
     name          TEXT              NOT NULL,
```

Comment:
I think there should be unique constraint on (rh_account_id, name) so customer can't have two baselines with the same name - they won't be able to distinguish them.

Reply:
I noticed the same yesterday while testing API! Nice, I agree :slightly_smiling_face: